src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -184,6 +184,27 @@ public function theLatestOrderHasAuthorizedPayment(OrderInterface $order, string
         Assert::eq($payment->getState(), $state);
     }
 
+    /**
+     * @Given the store has disabled :paymentMethodName payment method in Channel :channel
+     */
+    public function theStoreHasDisabledPaymentMethodInChannel($paymentMethodName, ChannelInterface $channel)
+    {
+        $paymentMethods = $this->paymentMethodRepository->findByName($paymentMethodName, 'en_US');
+
+        if (empty($paymentMethods)) {
+            throw new \InvalidArgumentException(sprintf('Payment method "%s" does not exist.', $paymentMethodName));
+        }
+
+        /** @var PaymentMethodInterface $paymentMethod */
+        $paymentMethod = $paymentMethods[0];
```

Comment:
```suggestion
     * @Given the store has disabled :paymentMethod payment method in Channel :channel
     */
    public function theStoreHasDisabledPaymentMethodInChannel(PaymentMethodInterface $paymentMethod, ChannelInterface $channel)
    {
```

I think you can do here the same like it's done here https://github.com/Sylius/Sylius/blob/1.14/src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php#L515-L524

Reply:
Here is how this step i linked is used somewhere else https://github.com/Sylius/Sylius/blob/1.14/features/shipping/applying_shipping_fee/applying_correct_shipping_fee_with_product_taxes_on_order.feature#L37

Reply:
ok